sfts/redshift_to_s3.py

```diff
@@ -181,7 +196,7 @@ def object_key_builder(key_prefix, *args):
     object_key = '_'.join(str(part) for part in key_parts)
     return object_key
 
-if 'start_date' in config and 'end_date' in config:
+if 'start_date' in config and 'end_date' in config and not dates:
```

Comment:
This may seem unnecessary, though I included "and not date" with future unified JSON schemas in mind.